packages/ui/react-ui-editor/src/components/TextEditor/codemirror.stories.ts

```diff
@@ -90,6 +90,7 @@ const setupSpace = async (count: number) => {
 };
 
 const count = 2;
+
 export const Default: StoryObj<{ id: number; client: Client; spaceKey: PublicKey }> = {
```

Comment:
Note that this was in a separate directory because it was specifically testing codemirror + ECHO performance w/o react. I think we may have lost the ability to run it during the storybook refactor last week (@thure) but it should probably be made clear somewhere that this story is quite different from the others. If not by code organization then by a prominent comment of some sort.